webrender/src/device/gfx/buffer.rs

```diff
@@ -75,7 +77,7 @@ impl<B: hal::Backend> Buffer<B> {
         non_coherent_atom_size_mask: u64,
     ) {
         let size = (data.len() * std::mem::size_of::<T>()) as u64;
-        let range = 0 .. ((size + non_coherent_atom_size_mask) & !non_coherent_atom_size_mask);
+        let range = 0 .. ((size + non_coherent_atom_size_mask) & !non_coherent_atom_size_mask).min(self.buffer_size as u64);
```

Comment:
why are we mapping more than necessary?

Reply:
The proper fix was using the correct alignment mask for uniform buffers. Fixed in: 79e2fa5bdd41a46b40082230c7f1813f982a3ca7